packages/bot-skeleton/src/scratch/hooks/tooltip.js

```diff
@@ -0,0 +1,70 @@
+import { blueInfo } from '../blocks/images';
+
+Blockly.Tooltip.HOVER_MS = 50;
+
+Blockly.Tooltip.show_ = () => {
+    let params;
+    Blockly.Tooltip.DIV = ((Blockly.Tooltip.poisonedElement_ = Blockly.Tooltip.element_), Blockly.Tooltip.DIV);
+    if (!Blockly.Tooltip.blocked_) {
+        Blockly.Tooltip.DIV.innerHTML = '';
+        for (params = Blockly.Tooltip.element_.tooltip; typeof params === 'function'; ) params = params();
+        params = Blockly.utils.string.wrap(params, Blockly.Tooltip.LIMIT);
+        params = params.split('\n');
+
+        params.forEach((param, index) => {
+            const div = document.createElement('div');
+            const text_span = document.createElement('span');
+
+            text_span.appendChild(document.createTextNode(param));
+            text_span.style.verticalAlign = 'middle';
+            Blockly.Tooltip.DIV.appendChild(div);
+
+            if (index === 0) {
+                const img = document.createElement('img');
+                const img_span = document.createElement('span');
+                img.src = blueInfo;
+                img.style.paddingRight = '8px';
+                img.style.verticalAlign = 'middle';
+                div.appendChild(img_span);
+                img_span.appendChild(img);
+            } else {
+                text_span.style.paddingLeft = '24px';
+            }
+
+            div.appendChild(text_span);
+        });
+
+        params = Blockly.Tooltip.element_.RTL;
```

Comment:
Just a suggestion, how about calling this direction?

```suggestion
        direction = Blockly.Tooltip.element_.RTL;
```